lib/parallel_tests/cli.rb

```diff
@@ -63,18 +63,24 @@ def run_tests_in_parallel(num_processes, options)
         groups.reject! &:empty?
 
         test_results = if options[:only_group]
-          groups_to_run = options[:only_group].collect{|i| groups[i - 1]}.compact
-          report_number_of_tests(groups_to_run)
-          execute_in_parallel(groups_to_run, groups_to_run.size, options) do |group|
-            run_tests(group, groups_to_run.index(group), 1, options)
-          end
-        else
-          report_number_of_tests(groups)
-
-          execute_in_parallel(groups, groups.size, options) do |group|
-            run_tests(group, groups.index(group), num_processes, options)
-          end
-        end
+           groups_to_run = options[:only_group].collect{|i| groups[i - 1]}.compact
+           report_number_of_tests(groups_to_run)
+           execute_in_parallel(groups_to_run, groups_to_run.size, options) do |group|
+             run_tests(group, groups_to_run.index(group), 1, options)
+           end
+         else
+           report_number_of_tests(groups)
+           if options[:features_from_process]
+             ran = []
+             for x in 0...num_processes
+               ran << File.expand_path(Dir.pwd) + "/features_from_process_#{x}.txt"
```

Comment:
idk why the expand_path is needed if it is added to pwd ...
prefer not dumping tempfiles into local directory it messes with git ...
can dump into `tmp/` maybe even use `Tempfile` to get a truly unique file